socorro/external/es/super_search_fields.py

```diff
@@ -4308,7 +4308,7 @@ def get(self):
         'form_field_type': None,
         'has_full_version': False,
         'in_database_name': 'uuid',
-        'is_exposed': False,
+        'is_exposed': True,
```

Comment:
This allows one to do a super search and filter on uuid. I talked to @adngdb and he said it should be fine. It wasn't exposed because it's pretty silly, but it made writing the CrashVerify endpoint a lot easier, so I exposed it.